src/js/fetching/fetch-movies.js

```diff
@@ -49,16 +43,17 @@ export const getPopular = async (page = 1) => {
       throw new Error(response.status);
     }
     const data = await response.json();
-    console.log('Poniżej przykladowy console.log dla popularnych');
     console.log(data);
-    // TO DO function here!
+
+    // TODO function here!
     renderMovies(data.results);
   } catch (error) {
     console.error(error);
   }
 };
 
 //  3.    ------ function fetch - get movies by title ------
+
```

Comment:
The comment must be on the line above the content being described (no break line)